torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -64,9 +64,11 @@ std::string getModulePath(Node* node) {
   return modulePath.substr(start, end - start);
 }
 
-c10::IValue getFunctionTuple(
+void getFunctionTuple(
```

Comment:
Just return a pair of IValue instead of passing a reference.

Reply:
And actually for debug_info_elements return `c10::optional`. It kind of makes it clear that it is optional and you get only when you have save_debug_info_in_bytecode set. Also rename `save_debug_info_in_bytecode` to `save_mobile_debug_info` since we are not saving it in bytecode.pkl.